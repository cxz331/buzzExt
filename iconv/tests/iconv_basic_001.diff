001+ string(16) "Zlutouck'y kun\n"
001- string(15) "Zlutoucky kun\n"