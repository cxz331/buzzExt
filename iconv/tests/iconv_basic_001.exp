string(15) "Zlutoucky kun\n"