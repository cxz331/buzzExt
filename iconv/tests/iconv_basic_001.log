
---- EXPECTED OUTPUT
string(15) "Zlutoucky kun\n"
---- ACTUAL OUTPUT
string(16) "Zlutouck'y kun\n"
---- FAILED
