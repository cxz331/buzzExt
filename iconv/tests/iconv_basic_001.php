<?php
setlocale(LC_ALL, "en_US.utf8");
$in_charset          = 'UTF-8';
$out_charset 	     = 'ASCII//TRANSLIT';
$string_to_translate = 'Žluťoučký kůň\n';

$string_out = iconv($in_charset, $out_charset, $string_to_translate);

var_dump($string_out);
?>
