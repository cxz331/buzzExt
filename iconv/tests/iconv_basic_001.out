string(16) "Zlutouck'y kun\n"